docs/cvl/storage-extension-annotations.md

```diff
@@ -0,0 +1,120 @@
+(storage-extension-annotations)=
+Automatic Storage-Extension Harnesses
+=====================================
+
+[Solidity and EIP-7201 namespace](https://eips.ethereum.org/EIPS/eip-7201)
+-----------------------------
+
+Upgradeable contracts frequently tuck new state variables into
+namespace “bookshelves” in storage:
+
+```solidity
+/** @custom:storage-location erc7201:my.project.book1 */
+struct Book1 { /* ... */ }
+```
```

Comment:
I would prefer we use the example from the EIP _or_ the vault example that you have below.